page01_index.php

```diff
@@ -62,117 +99,132 @@
 		</div>
 
 
+	
+		<!-- image background -->
+		<div class="w3-orange container w3-image">
+			<div class="bg">
 
-		<!-- background container; colour orange -->
-		<div class="container2" style="max-width:100%">
 
+				<br>
+				<br>
+				<br>
+				<br>
 
-			<div class="w3-panel w3-padding-large w3-margin-left w3-margin-right">
 
+				<center>
+					<div class= "container w3-panel w3-padding-large" style="height:80%;width:80%; background-color:rgba(255, 255, 255, .2);">
```

Comment:
clear container to keep content together and also have an aesthetic concept - matching my 3.2 design